solutions/fully-configurable/catalogValidationValues.json.template

```diff
@@ -4,6 +4,5 @@
   "kms_encryption_enabled_buckets": true,
   "region": "us-south",
   "prefix": $PREFIX,
-  "existing_cos_instance_crn": $COS_INSTANCE_CRN,
-  "existing_resource_group_name": "geretain-test-resources"
+  "existing_cos_instance_crn": $COS_INSTANCE_CRN
```

Comment:
Can we set `enable_activity_tracker_event_routing_to_cloud_logs` to true here so its done as part of DA validation?